contrib/export/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/cosmos/cosmos-sdk/x/genutil"
+	"github.com/tendermint/tendermint/types"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+var (
+	migrationMap = extypes.MigrationMap{
+		"036": v036.Migrate,
```

Comment:
```suggestion
		"v0.36": v036.Migrate,
```

Seems like this will provide a better UX, no?